src/main/java/com/thealgorithms/others/Sudoku.java

```diff
@@ -70,12 +90,11 @@ public static boolean solveSudoku(int[][] board, int n) {
             return true;
         }
 
-        // Else for each-row backtrack
+        // Try placing numbers 1 to 9 in the empty cell
```

Comment:
And here also

Reply:
Changed this comment to: `// Try placing numbers 1 to n in the empty cell (typically n=9)`